src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
+  if (indexOfTime >= 0) {
+    newDate = newDate.substring(0, indexOfTime);
+  }
+  return newDate;
+}
+
 export function formatDateLong(date) {
   return moment(date).format('MMMM DD, YYYY');
 }
 
+export function formatDateWithoutTimeLong(date) {
```

Comment:
We're not using `formatDateLong` anywhere in GIBS, but as this function exists in a common helper file, I'm leaving it in case other apps use or plan to use it in the future. They may want a generic function that doesn't drop time and timezone from the date.